packages/core-ethereum/src/index.ts

```diff
@@ -121,7 +121,12 @@ export default class HoprEthereum {
     return useCache ? this.cachedGetNativeBalance() : this.uncachedGetNativeBalance()
   }
 
-  public smartContractInfo(): string {
+  public smartContractInfo(): {
```

Comment:
Worth extracting a type to reduce code duplication?

Reply:
I initially tried without any types but typescript complained that it required it to be typed

Reply:
I mean `declare type ContractInfo = {...}` which you can reuse here and in line 593